types/applepayjs/index.d.ts

```diff
@@ -326,9 +326,9 @@ declare namespace ApplePayJS {
         paymentTiming?: ApplePayPaymentTiming;
 
         /**
-         * The [ISO 8601 formatted] date of the first payment.
+         * The date of the first payment.
          */
-        recurringPaymentStartDate?: string;
+        recurringPaymentStartDate?: Date;
```

Comment:
As these are breaking changes to the interface, if these type definitions use SemVer then the version of the type definitions should be updated.

Looks like the latest version is [14](https://developer.apple.com/documentation/apple_pay_on_the_web/apple_pay_on_the_web_version_history?changes=latest_major) now.

Reply:
hey, sorry, didn't get your idea, probably because I'm not used to contribute to that kind of project.
looking ApplePaySession class, there is the following comment:
```
@param version - The version number of the ApplePay JS API you are using. The current API version number is 14.
```

should I update any other place?

Reply:
No, I meant the [npm package version](https://www.npmjs.com/package/@types/applepayjs?activeTab=versions), but it looks like someone updated it to v14 last year.

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/2a00dd1bfb0153399653598a2bc74969929c13da/types/applepayjs/package.json#L4

Reply:
Ah ok, great 🙂